README.md

```diff
@@ -0,0 +1,399 @@
+# `turludock` - Robot Operating System (ROS) Docker Containers with X11 and Wayland support for Linux
+[![N|Solid](http://turlucode.com/wp-content/uploads/2017/10/turlucode_.png)](http://turlucode.com/)
+
+`turludock` aims to provide different versions of ROS as docker containers, but with GUI 
+support! This means your local OS is no more bound to the version of ROS you are using! 
+You can use _any_ version of ROS with _any_ Linux distribution, thanks to the amazing 
+power of docker!
+
+# TL;DR
+## Build image
+```sh
+# Install tool
+pip install turludock
+
+# Build ROS image from presets (turludock which presets)
+turludock build -e noetic_mesa
+# Build from manual config 
+turludock build -c custom.yaml --tag custom_tag
+
+# Check supported versions
+turludock which ros
+turludock which cuda ROS_CODENAME
+```
+All commands have a `--help` support.
+> [Example YAML config](examples/noetic_nvidia_custom.yaml)
+
+## Run container
+[See here](#running-the-image-as-current-user) how to run a container.
+
+# Getting Started
+The idea is to have HW accelerated GUIs with docker. Generally it has proven that this is 
```

Comment:
This part you might want to move to the top, because the aim is to have Docker containers with HW accelerated GUIs for any version of ROS.